docker/Dockerfile

```diff
@@ -41,17 +40,22 @@ COPY requirements ./requirements
 
 # Install build dependencies, then Python requirements, then remove build dependencies
 # to reduce the resulting image size
-RUN apk update && apk add --virtual build-deps make gcc g++ musl-dev apache2-dev && \
-  apk add --virtual build-headers postgresql-dev \
-  jpeg-dev \
-  zlib-dev \
-  openjpeg-dev \
-  libevent-dev && \
-  cd /app && \
-  pip3 install -r requirements/base.txt -r requirements/docker.txt && \
-  apk del build-deps build-headers
-
-RUN adduser -S www -u 1000 && chown -R www /app
+RUN <<EOF
+    set -ex
+    apt-get update -qq
+    apt-get install -qq postgresql-client netcat-openbsd $APT_BUILD_DEPS
```

Comment:
Why install `netcat-openbsd`?

Reply:
The `wait-for` script [calls it internally](https://github.com/mrako/wait-for/blob/a7411a6ae38ccc4b5916332a29ca94756db6f037/wait-for#L66). On Alpine, `nc` was available through busybox; on Debian, we have to install it.